oxidation/crates/parsec_client_types/src/local_manifest.rs

```diff
@@ -324,6 +324,24 @@ impl LocalFileManifest {
         reference.version = remote_manifest.version;
         reference == *remote_manifest
     }
+
+    pub fn evolve_and_mark_updated(
+        mut self,
+        size: u64,
```

Comment:
Why these fields in particular ?
I haven't implemented this method in Rust because an update method can take a lot of args and this method brings no more

Reply:
Because I needed to update those fields in `prepare_write`. I wanted to stick to the original implementation for the moment but we can definitely change it in the future.